src/network/networkd.c

```diff
@@ -133,10 +133,8 @@ int main(int argc, char *argv[]) {
         }
 
         r = manager_rtnl_enumerate_rules(m);
-        if (r < 0) {
-                log_error_errno(r, "Could not enumerate rules: %m");
-                goto out;
-        }
+        if (r < 0)
```

Comment:
Shouldn't it check for ENOSYS or another particular return value instead? It'd probably be nicer to put this check into the first call inside of `m_r_enumerate_rules`, and simply return 0 from that function in that case (and not even warn, just log it at debug level, because if the support is compiled out in the kernel, then there are no rules, so just doing nothing is exactly correct).

Reply:
> Shouldn't it check for ENOSYS or another particular return value instead?

If routing policy rules are not supported, the netlink calls return `-EOPNOTSUPP`.

Reply:
updated